### DISCUSSION TITLE
关于OpenWrt-Packages仓库的来源问题

### DISCUSSION THREAD

**Discussion by black-binary** [Boos4721/openwrt#97] [Q&A/答疑解惑] [answered]
我观察到[OpenWrt-Packages](https://github.com/Boos4721/OpenWrt-Packages)有机器人在更新代码，但是似乎没有看到github actions的部署，比较好奇这个代码更新是如何进行的。以及想问，如果想自行移植官方的package，具体应该如何做呢？感谢

**Comment by Boos4721** [comment, marked as answer]
it is a synchronization `SHELL` script for another CI, like `git clone package` and `svn co package`.
